docs/standalone.md

```diff
@@ -100,7 +100,6 @@ your own pre-compiled assets, for example, via a CDN.
 1. Optionally, add TwirlKeys.templateImports in build.sbt:
     ```sbt
         TwirlKeys.templateImports ++= Seq(
-          "uk.gov.hmrc.govukfrontend.views.html.components._",
-          "uk.gov.hmrc.govukfrontend.views.html.helpers._"
```

Comment:
Removed this useless import statement. It was only bringing in the auto-compiled `formWithCSRF` that should be used via the upper-cased alias `uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF`